package.json

```diff
@@ -39,6 +40,7 @@
   "devDependencies": {
     "chai": "^3.5.0",
     "codecov-lite": "^0.1.3",
+    "glob": "^7.1.1",
     "mocha": "^3.0.2",
```

Comment:
This should be a dependency, not a dev dependency (also make sure the yarn.lock file is updated)